ethik/query.py

```diff
@@ -146,7 +146,10 @@ def from_taus(
 
         q_mins = quantiles.loc[q[0]].to_dict()
         q_maxs = quantiles.loc[q[1]].to_dict()
-        means = X_test.mean().to_dict()
+        # Cannot use `means = X_test.mean().to_dict()` as it may give a different
+        # result depending on whether the feature is alone in the dataset or not
+        #  Instead, we compute the means for each pandas series independently
+        means = {feature: X_test[feature].mean() for feature in X_test.columns}
```

Comment:
Do you have an example of this?

Reply:
I wrote a test. Otherwise, in #126  :)

Reply:
Do you mean I should put it in the comment?

Reply:
No not at all, it's just for my general knowledge! I saw your issue, I just haven't had time to look at it. We're good to go!